module/zfs/vdev_raidz.c

```diff
@@ -135,6 +141,22 @@
 	VDEV_RAIDZ_64MUL_2((x), mask); \
 }
 
+/*
+ * For testing only: logical offset at which to pause the expansion.
+ */
+unsigned long raidz_expand_max_offset_pause = 0;
+
+/*
+ * Maximum amount of copy io's outstanding at once.
+ */
+unsigned long raidz_expand_max_copy_bytes = 10 * SPA_MAXBLOCKSIZE;
+
+/*
+ * Apply raidz map abds aggregation if the number of rows in the map is equal
+ * or greater than the value below.
+ */
+unsigned long raidz_io_aggregate_rows = 4;
```

Comment:
all of these can be `static` AFAICT

Reply:
The other module parameters that I checked were not declared `static`.  This might be a holdover from the illumos codebase where `ZFS_MODULE_PARAM` is a no-op, so if the variable is not changed by the C code, the compiler could optimize away the load instructions, instead assuming that it always has the value set at initialization.  It's possible that ZFS_MODULE_PARAM() prevents that optimization on Linux/FreeBSD, so we can change this to declare all module parameters as `static`.  But for this PR I'd rather follow the existing convention.

Reply:
Ah, I was looking at my tree.  When I rebase I'll be sure to change this to follow current conventions, thanks for pointing it out.